File: samples/dynamic-query-groovy/src/main/java/io/dscope/camel/snowflake/sample/groovy/SampleMain.java
package io.dscope.camel.snowflake.sample.groovy;

import org.apache.camel.main.Main;

public class SampleMain {
    public static void main(String[] args) throws Exception {
        EnvLoader.load();
    Main main = new Main();
    main.configure().addRoutesBuilder(new DynamicQueryRoute());
        main.run(args);
    }
}

File: samples/dynamic-query-kotlin/src/main/kotlin/io/dscope/camel/snowflake/sample/kotlin/SampleMain.kt
package io.dscope.camel.snowflake.sample.kotlin

import org.apache.camel.main.Main

fun main(args: Array<String>) {
    EnvLoader.load()
    val main = Main()
    main.configure().addRoutesBuilder(DynamicQueryRoute())
    main.run(args)
}
